src/plots/cartesian/axes.js

```diff
@@ -1163,19 +1165,21 @@ function formatCategory(ax, out) {
     var tt = ax._categories[Math.round(out.x)];
     if(tt === undefined) tt = '';
     out.text = String(tt);
+}
 
-    // Setup ticks and grid lines boundaries
-    // at 1/2 a 'category' to the left/bottom
-    if(ax.tickson === 'boundaries') {
-        var inbounds = function(v) {
-            var p = ax.l2p(v);
-            return p >= 0 && p <= ax._length ? v : null;
-        };
+function formatMultiCategory(ax, out, hover) {
+    var v = Math.round(out.x);
+    var cats = ax._categories[v] || [];
+    var tt = cats[1] === undefined ? '' : String(cats[1]);
+    var tt2 = cats[0] === undefined ? '' : String(cats[0]);
 
-        out.xbnd = [
-            inbounds(out.x - 0.5),
-            inbounds(out.x + ax.dtick - 0.5)
-        ];
+    if(hover) {
+        // TODO is this what we want?
+        out.text = tt2 + ' - ' + tt;
```

Comment:
This looks like:

![peek 2018-12-05 19-02](https://user-images.githubusercontent.com/6675409/49552146-4df34d00-f8c0-11e8-9c44-3d212e66d242.gif)


I could use a second opinion here.

Reply:
in-person @alexcjohnson mentioned that a simple space ` ` could perhaps be better than the current ` - ` . Thoughts?

Reply:
I still think a single space is my vote, but it's not a big deal to me either way. @nicolaskruchten @chriddyp do you have a preference?

Reply:
... and note that `hovertemplate` _just work_<sup>TM</sup> with multicategory axes:

https://github.com/plotly/plotly.js/blob/a25c5e3cf23bd5af3f2bf45e3f161d7569fdf9fd/test/jasmine/tests/hover_label_test.js#L2096

so one can easily customize the appearance.

Reply:
nice re hovertemplates! I favour the hyphen personally